tests/unit/test__fixture__helpers.py

```diff
@@ -0,0 +1,40 @@
+import os
+from unittest.mock import MagicMock
+
+from grasshopper.lib.fixtures import _get_tagged_scenarios
+
+
+def test_get_tagged_scenarios_happy():
```

Comment:
ideally, we'd also use the `caplog` fixture here and check that the correct message was output. there is some code for validating specific messages (`was_message_logged`) in the `unit/conftest.py`

probably there are other tests in this set where that would be appropriate as well.